### ISSUE TITLE
Datepicker is not working on any page with Norwegian language

### ISSUE THREAD

**Issue by Git07Omkar** [DNNCommunity/DNN.Events#201]
I have to use this module in DNN 9.3.1 and in the Norwegian language, which follows the "22.10.19" date pattern.
It seems like this issue is only for the Norwegian language, as its working fine for American English which follows the "22/10/19" date pattern.
Please suggest.
Thanks in advance

**Comment by thabaum** [comment]
I believe a new datepicker is being created to be used in DNN.  Would there need to be a culture code setting added to DNN Events for Telerik to work here?

https://www.telerik.com/forums/the-value-dd-mm-yyyy-not-valid

A similar issue I believe with a similar UI tool.

**Comment by Git07Omkar** [comment]
Do you know the expected time for the new release of event module with the culture code setting?

**Comment by EPTamminga** [comment]
If time allows, and the PR works correctly, I will create a new release in the upcoming week. 
@thabaum Tnx for the fixes!

**Comment by thabaum** [comment]
Thank you all so much for the contributions and time helping DNN Events @EPTamminga You are great assets to our community team!

**Comment by Git07Omkar** [comment]
Is this issue fixed ?

**Comment by EPTamminga** [comment]
No new release created. I was working on some other issues. When time allows, I will regroup myself and have a go at a new release. Sorry for the delay.

**Comment by thabaum** [comment]
@Git07Omkar have you tried the new release 7.0.5?  Thanks @EPTamminga

**Comment by Git07Omkar** [comment]
@thabaum Yes, I have tried the latest version 7.0.6 of this module in DNN 9.5.0, But still got the same issue.
